scrawls/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+from rest_framework import generics, permissions
+from rest_framework.response import Response
+from rest_framework.views import status
 
-# Create your views here.
+from .serializers import WallsSerializer
+from .models import Wall, Comment
+
+
+class CreateWall(generics.CreateAPIView):
+    permission_classes = (permissions.AllowAny,)
+
+    def post(self, request, *args, **kwargs):
+        name = request.data.get("name", "")
+        address = request.data.get("address", "")
+        lat = request.data.get("lat", "")
+        lng = request.data.get("lng", "")
+        range = request.data.get("range", "")
```

Comment:
Is `range` a reserved word in python? Curious why it's showing up in blue here...

Reply:
Good question, I don't think so, but maybe. It's working at the moment but there's probably something to look into there.